### PR TITLE
Add a pkgdown site

### PR DESCRIPTION
**Pull request by hadley**
This uses the dev version of pkgdown because I've been working on it lately (and it looks much nicer/is more accessible); it'll be released in the next week or two so can revert to the CRAN version then.

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- title: Check messages
+  contents:
+  - starts_with("check_")
+
+- title: Internal helpers
+  contents:
+  - get_message_data
+  - write_po_file
+
+authors:
```
Comment (MichaelChirico) [inline _pkgdown.yml:25]: should you be here too? or is this just "maintainers"

**Review by MichaelChirico — APPROVED** [review]

File: _pkgdown.yml (hunk 1 above)
Comment (hadley) [reply _pkgdown.yml:25]: Shamefully I hard coded my website into pkgdown itself 😅

**Comment by hadley** [comment]
You might also need to do some stuff on the "pages" setting page, and you'll need to edit the url on the repo home page to point to https://michaelchirico.github.io/potools

**Comment by MichaelChirico** [comment]
Looks like GitHub automatically detected something, the page was already ready when I checked just now :tada: 

Thanks again!

**Pull request merged**